src/scripts/components/VideoStatus.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import React, { Component } from 'react'
+// import styled from 'styled-components'
+// import Button from './foundations/Button'
+// import Input from './foundations/Input'
+import Wrapper from './foundations/Wrapper'
+
+type Props = {
+  video: Object,
```

Comment:
Can this instead be a `Video` record? That is what the redux store would contain anyways (and it looks like that is the more accurate type in this case already)